kong/router/expressions.lua

```diff
@@ -1,11 +1,16 @@
 local _M = {}
 
 
+local re_gsub = ngx.re.gsub
+
+
 local atc = require("kong.router.atc")
 local gen_for_field = atc.gen_for_field
 
 
 local OP_EQUAL    = "=="
+local NET_PORT_REG = [[(net\.port)(\s*)([=><!])]]
```

Comment:
This regex doesn't cause any bug now, but it might become a bug with more and more String-type fields are added.

Is it possible to add a regression test to prevent that?

Reply:
`net.port` is a undocumented feature, I think that now there is few people using it, and in next major version we will remove this field.